graylog2-web-interface/src/pages/SecurityPage.tsx

```diff
@@ -16,60 +16,74 @@
  */
 
 import * as React from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
-import { IfPermitted } from 'components/common';
-import { Alert } from 'components/bootstrap';
-import { Link } from 'components/common/router';
-import Routes from 'routing/Routes';
-import AppConfig from 'util/AppConfig';
+import { Col, Row } from 'components/bootstrap';
+import { DocumentTitle, PageHeader, Spinner } from 'components/common';
+import { NodesStore } from 'stores/nodes/NodesStore';
+import PluginList from 'components/enterprise/PluginList';
+import ProductLink from 'components/enterprise/ProductLink';
+import { GraylogClusterOverview } from 'components/cluster';
+import HideOnCloud from 'util/conditional/HideOnCloud';
+import { useStore } from 'stores/connect';
+import EnterpriseProductLink from 'components/enterprise/EnterpriseProductLink';
 
-const StyledH2 = styled.h2`
-  font-weight: bold;
-  margin-bottom: 15px;
-`;
-
-const StyledH4 = styled.h4`
-  font-weight: bold;
+const StyledHeader = styled.h2`
   margin-bottom: 10px;
 `;
 
-const StyledAlert = styled(Alert)`
-  margin-top: 15px;
-`;
+const BiggerFontSize = styled.div(({ theme }) => css`
+  font-size: ${theme.fonts.size.large};
+`);
 
-const isCloud = AppConfig.isCloud();
+const SecurityPage = () => {
+  const nodes = useStore(NodesStore);
 
-const LinkTo = () => {
-  if (Routes.pluginRoute('SYSTEM_LICENSES', false)) {
-    return (
-      <IfPermitted permissions="licenses:create">
-        <p>
-          See <Link to={Routes.pluginRoute('SYSTEM_LICENSES')}>Licenses page</Link> for details.
-        </p>
-      </IfPermitted>
-    );
+  if (!nodes) {
+    return <Spinner />;
   }
 
-  return (
-    <p>
-      Please see <a href="https://www.graylog.org/products/enterprise" rel="noopener noreferrer" target="_blank">our product page</a> for details.
-    </p>
-  );
-};
+  const { clusterId } = nodes;
 
-const SecurityPage = () => {
   return (
-    <StyledAlert bsStyle="danger" className="tm">
-      <StyledH2>Invalid License for the Security plugin</StyledH2>
-      <StyledH4>Security plugin is disabled</StyledH4>
-      <p>
-        The security plugin is disabled because a valid Graylog for Security license was not found{Routes.pluginRoute('SYSTEM_LICENSES', false) ? '' : ' and the enterprise plugin is not installed'}.
-      </p>
-      {isCloud
-        ? (<>Contact your Graylog account manager.</>)
-        : (<LinkTo />)}
-    </StyledAlert>
+    <DocumentTitle title="Try Graylog Security">
+      <div>
+        <PageHeader title="Try Graylog Security">
+          {null}
+          <span>
+            Extend Graylog's capabilities for detecting, investigating, and responding to cybersecurity
```

Comment:
Let's go with the review bot suggestion here it is technically a best practice albeit a nit.